rectools/models/nn/transformers/base.py

```diff
@@ -601,20 +602,34 @@ def __setstate__(self, state: tp.Dict[str, tp.Any]) -> None:
         self.__dict__.update(loaded.__dict__)
 
     @classmethod
-    def load_from_checkpoint(cls, checkpoint_path: tp.Union[str, Path]) -> tpe.Self:
-        """
-        Load model from Lightning checkpoint path.
+    def load_from_checkpoint(
+        cls,
+        checkpoint_path: tp.Union[str, Path],
+        map_location: tp.Union[str, torch.device, None] = None,
+        config_update: tp.Dict[str, tp.Any] = {},
```

Comment:
Mutable objects shouldn't be default argument values. Please always pass ``None`` as default value in such cases.